diff --git a/CMakeLists.txt b/CMakeLists.txt
index 78ab556..76b5019 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,5 @@
 #The CMake Minimum version that is required.
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.5)

 set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/;${CMAKE_MODULE_PATH}")
 message(STATUS "Module path is now set to: " ${CMAKE_MODULE_PATH} )
